docs/reference/commandline/dockerd.md

```diff
@@ -856,29 +854,23 @@ PING host.docker.internal (192.0.2.0): 56 data bytes
 ### Miscellaneous options
 
 IP masquerading uses address translation to allow containers without a public
-IP to talk to other machines on the Internet. This may interfere with some
-network topologies and can be disabled with `--ip-masq=false`.
+IP to talk to other machines on the internet. This may interfere with some
+network topologies, and can be disabled with `--ip-masq=false`.
 
-Docker supports softlinks for the Docker data directory (`/var/lib/docker`) and
+Docker supports soft links for the Docker data directory (`/var/lib/docker`) and
 for `/var/lib/docker/tmp`. The `DOCKER_TMPDIR` and the data directory can be
 set like this:
 
-```console
-$ DOCKER_TMPDIR=/mnt/disk2/tmp /usr/local/bin/dockerd --data-root /var/lib/docker -H unix:// > /var/lib/docker-machine/docker.log 2>&1
-```
-
-or
-
 ```console
 $ export DOCKER_TMPDIR=/mnt/disk2/tmp
 $ /usr/local/bin/dockerd --data-root /var/lib/docker -H unix:// > /var/lib/docker-machine/docker.log 2>&1
```

Comment:
We can probably drop the full path for these and just use `dockerd` (showing the full path doesn't seem to add value)